rust/private/clippy.bzl

```diff
@@ -189,7 +189,8 @@ $ bazel build --aspects=@rules_rust//rust:defs.bzl%rust_clippy_aspect \
 )
 
 def _rust_clippy_rule_impl(ctx):
-    files = depset([], transitive = [dep[OutputGroupInfo].clippy_checks for dep in ctx.attr.deps])
+    clippy_ready_targets = [dep for dep in ctx.attr.deps if "clippy_checks" in dir(dep[OutputGroupInfo])]
+    files = depset([], transitive = [dep[OutputGroupInfo].clippy_checks for dep in clippy_ready_targets])
```

Comment:
Just checking, these changes necessary for the clippy work here? I may have missed if this is building on another PR, so ignore this, if that's the case.

Reply:
Oh, this isn't intended for this PR. good catch.